posts/Ajay/2025-11-03-introducing-the-mongodb-extension-for-hibernate-orm.adoc

```diff
@@ -0,0 +1,72 @@
+= Introducing the MongoDB Extension for Hibernate ORM (Public Preview)
+Ajay Tandon
+:awestruct-tags: ["MongoDB", "Hibernate", "ORM", "JPA", "Java"]
```

Comment:
```suggestion
:awestruct-tags: ["MongoDB", "Hibernate ORM", "JPA"]
```

`Hibernate` and `Java` would match everything on the site, so they are not needed. The `ORM` tag is not used on this site, we use `Hibernate ORM` instead.